docs/monthly-meeting/2023-05.md

```diff
@@ -0,0 +1,121 @@
+# Documentation Community Team Meeting (May 2023)
+
+- **Date:** 2023-05-01
+- **Time:** [19:00 UTC](https://arewemeetingyet.com/UTC/2023-05-01/19:00/Docs%20Meeting) 
+- **This HackMD:** https://hackmd.io/@encukou/pydocswg1
+- [**Discourse thread**](https://discuss.python.org/t/26182) (for May)
+- [**Meeting reports**](https://docs-community.readthedocs.io/en/latest/monthly-meeting/index.html) (the latest one might be an [**unmerged PR**](https://github.com/python/docs-community/pulls))
+- **Calendar event:** (send your e-mail to Mariatta for an invitation)
+- **How to participate:**
+  -  Go to [Google Meet](https://meet.google.com/dii-qrzf-wkw) and ask to be let in.
+  -  To edit notes, click the “pencil” or “split view” button on the [HackMD document](https://hackmd.io/@encukou/pydocswg1). You need to log in (e.g. with a GitHub account).
+
+By participating in this meeting, you are agreeing to abide by and uphold the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/).
+Please take a second to read through it!
+
+
+## Roll call
+
+(Name / `@GitHubUsername`)
+- Mariatta `@Mariatta`
+- Petr Viktorin `@encukou`
+- Cristián Maureira-Fredes `@cmaureir`
+- Hugo van Kemenade`@hugovk`
+- Jim DeLaHunt `@JDLH`
+- Ryan Duve `@ryan-duve`
+- Joanna Jablonski `@jablonskidev`
+- Ezio Melotti `@ezio-melotti`
+- Keto Zhang `@ketozhang`
+- Manuel Kaufmann `@humitos`
+- Ned Batchelder `@nedbat`
+- C.A.M. Gerlach `@CAM-Gerlach`
+- Marcus Sherman `@betteridiot`
+- Yaa Nuamah Kusi-Fordjour `@`
+
+
+## Introductions
+
+> If there are any new people, we should do a round of introductions.
+
+- Petr, Python core dev, runs these meetings
+- Mariatta, one of the people who started the Docs group. Just chaired Pycon US
+- Hugo, from Helsinki Python core dev
+- Ryan Duve, data scientist, Python user interested helping with community docs stuff
+- Joanna, Python education
+- Jim, from Vancouver, Canada; reads docs & fixes things he notices
+- Manuel Kaufmann from Argentina now in Spain, translations, Read the Docs, CPython translation in Spanish, PSF Fellow
+- Marcus Sherman, from Portland, Maine @ Northeastern University, `@betteridiot`, biologist by training
+- Ned, from Boston, made the docs official, eager to see this group grow into a large & welcoming community
+- Cristián Maureira-Fredes, CPython Spanish translation
+- Keto Zhang, software engineer in astronomy, went to Cam's talk at PyCon
+- CAM, gave the talk at PyCon, new Python core dev, from Huntsville, Alabama, Spyder maintainer, looking to be a full-time open-source dev & documentarian
+- Ezio, core Dev, working on Docs and infrastucture
```

Comment:
```suggestion
- Ezio, core Dev, working on Docs and infrastructure
```

Reply:
Also may as well be consistent with caps, too?

```suggestion
- Ezio, core Dev, working on docs and infrastructure
```